uploader/src/main/kotlin/com/cloudinary/upload/Uploader.kt

```diff
@@ -153,6 +153,9 @@ class Uploader internal constructor(val cloudinary: Cloudinary, clientFactory: H
     }
 
     companion object {
+        /**
+         * TODO Internal
+         */
```

Comment:
?

Reply:
oh... that's the way to ignore in the reference??? ichsa!

Reply:
No that's actually a real TODO.
This PR is a general API cleanup, this is a part of the API that still needs a bit of work, however since it's uploader it's not for this phase (so - TODO).